### ISSUE TITLE
how can H750 support compiling complete firmware when using external storage?

### ISSUE THREAD

**Issue by suiaiyun** [ArduPilot/ardupilot#24330]
## Feature request

**Platform**
[ * ] All
[  ] AntennaTracker
[  ] Copter
[  ] Plane
[  ] Rover
[  ] Submarine

Hello, how can H750 support compiling complete firmware when using external storage? FLASH_ SIZE_KB  When is set to 128, the compiled firmware is not complete. What can I do to obtain complete firmware?

**Comment by andyp1per** [comment]
This is a support request - closing.

Please look at one of the boards that does this e.g. SPRacingH7

**Comment by suiaiyun** [comment]
> 这是一个支持请求 - 关闭。
> 
> 请查看执行此操作的主板之一，例如 SPRacingH7

Perhaps my description was not detailed enough, let me rephrase the problem I encountered:

1. When 'FLASH_ SIZE_ KB 128' is configured in hwdef.dat, the compiled firmware does not include Script function, and SRC_ENABLE cannot be found in the parameter list_ ENABLE parameter.

2. When 'define HAL_USE_EMPTY_STORAGE 1' is configured in hwdef.dat, all modifications made to the parameters will not be saved and will be restored to their original state after restarting. If this line is not added, the parameter is saved in a file called arduplane. stg on the SD card, and if the SD card flight control is not installed, it cannot run.

Thank you to every developer for their efforts.

**Comment by andyp1per** [comment]
1. Can you force scripting enable in the hwdef?
2. Yes, you have to use SD card for parameter storage
